pkg/util/mount/mount.go

```diff
@@ -94,13 +94,23 @@ func (mounter *SafeFormatAndMount) FormatAndMount(source string, target string,
 }
 
 // New returns a mount.Interface for the current system.
-func New(mounterPath string) Interface {
+func New() Interface {
+	return &Mounter{}
+}
+
+// NewCustomMounter returns a mount.Interface for the current system.
+// It provides options to override the default mounter behavior.
+func NewCustomMounter(mounterPath, mounterRootfsPath string) Interface {
 	// If mounter-path flag is not set, use default mount path
-	if len(mounterPath) == 0 {
+	if mounterPath == "" {
 		mounterPath = mount
 	}
+	if mounterRootfsPath == "" {
+		mounterRootfsPath = "/"
+	}
 	return &Mounter{
-		mounterPath: mounterPath,
+		mounterPath:       mounterPath,
+		mounterRootfsPath: mounterRootfsPath,
```

Comment:
`/` to kubelet is `/rootfs` to the mounter container. So if kubelet wants to mount `/foo/bar`, then it has to pass `/rootfs/foo/bar` to the container.